app/v1/services/DeleteEmploymentExpensesService.scala

```diff
@@ -41,7 +40,7 @@ class DeleteEmploymentExpensesService @Inject() (deleteEmploymentExpensesConnect
       logContext: EndpointLogContext,
       correlationId: String): Future[DeleteEmploymentExpensesServiceOutcome] = {
 
-    val result =  EitherT(deleteEmploymentExpensesConnector.deleteEmploymentExpenses(request)).leftMap(mapDownstreamErrors(errorMap))
+    val result = EitherT(deleteEmploymentExpensesConnector.deleteEmploymentExpenses(request)).leftMap(mapDownstreamErrors(errorMap))
```

Comment:
appreciate this was just a reformatting, but while you're there, the code can be simplified to:
```
deleteEmploymentExpensesConnector.deleteEmploymentExpenses(request).map(_.leftMap(mapDownstreamErrors(errorMap)))
```
(i.e. remove the EitherT & the val result)